apps/barista/src/components/live-example/live-example.html

```diff
@@ -4,12 +4,16 @@
   <div #demoPlaceholder></div>
 </div>
 
-<div class="ba-live-example-actions">
+<div
+  class="ba-live-example-actions"
+  [class.ba-live-example-source-hidden]="_sourceHidden"
```

Comment:
Can we just `ngIf` here instead of applying a hidden class?

Reply:
I use `ngIf` for the source itself in line 65, but here I need a class for some minor styling updates when the source-block is hidden.